spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -36,6 +37,7 @@
   end
 
   context 'when target params is missing' do
```

Comment:
Because you used the context `'when the request is valid'`, you could use another context `'when the request is invalid'` and within it another one called `'with logged-in user'`, so that you can omit the `let(:headers) { auth_headers }` in each context of the failing requests